clang/test/CXX/drs/cwg7xx.cpp

```diff
@@ -71,6 +76,17 @@ namespace cwg712 { // cwg712: partial
 #endif
 }
 
+namespace cwg713 { // cwg713: yes
```

Comment:
Please check how just released Clang 19.1-rc1 handles this test, and choose between 19 and 20 accordingly.